cmd/opampsupervisor/e2e_test.go

```diff
@@ -1261,8 +1261,8 @@ func createBadCollectorConf(t *testing.T) (*bytes.Buffer, []byte) {
 	return bytes.NewBuffer(colCfg), h.Sum(nil)
 }
 
-func createHealthCheckCollectorConf(t *testing.T) (cfg *bytes.Buffer, hash []byte, remotePort int) {
-	colCfgTpl, err := os.ReadFile(path.Join("testdata", "collector", "healthcheck_config.yaml"))
+func createHealthCheckCollectorConf(t *testing.T, nopPipeline bool) (cfg *bytes.Buffer, hash []byte, remotePort int) {
```

Comment:
Just so I understand, what's the change in behavior that required this change?

Reply:
Thanks, I think I see the issue now. This probably warrants a second look at some point, but if the tests are passing I think we're okay for the moment.